CustomExceptions.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CUSTOM_EXCEPTIONS_H
+#define CUSTOM_EXCEPTIONS_H
+
+#include <exception>
+#include <string>
+
+class InvalidInputException : public std::exception {

```

Comment:
using this std:: 
use using namespace std that is more readable.

Reply:
Done